data/human/intro missions.txt

```diff
@@ -508,3 +508,95 @@ conversation "end of intro missions"
 		
 	label end
 	`	He hands you a credit chip worth <payment>, and says, "Best of luck to you, Captain. Don't hesitate to look me up if you're ever in the area. And remember to check the spaceports wherever you land. You never know who you'll find wandering around there."`
+
+
+
+mission "Intro [Mining]"
+	landing
+	name "Explore Mining"
+	description "Learn about mining."
+	source "Sunracer"
+	to offer
+		or
+			has "ships: Interceptor"
+			has "ships: Drone"
+			has "ships: Fighter"
+			has "ships: Light Warship"
+			has "ships: Medium Warship"
+			has "ships: Heavy Warship"
+			has "ships: Superheavy"
+	
+	on offer
+		conversation
+			`	On your approach to <planet> you notice a large warehouse with private landing pads for the Clementine Metals Company. Someone there would know about mining asteroids. Would you like to ask them?`
+			choice
+				`	It would be nice to get more information.`
+					goto mining
+				`	I'm busy now, but maybe later.`
+					defer
+				`	I'm not interested.`
+					decline
+	
+			label "mining"
+			`	The warehouse is surprisingly well organized for a place that deals with raw ores. The receptionist tells you that the owner is due back shortly and points you to some picnic tables outside. You have a seat and watch the trucks loading and carting off various minerals. Soon enough a battered Aerie so covered in scorch marks and hull repairs that you can barely make out it's registration markings settles in next to you. When the hatch opens, a woman with close cut hair and a worn but crisp navy flight suit walks up to you. "I'm Captain Zinaida she says without preamble. You were wanting to asking about mining?" You nod your head and she says, "Well, go ahead."`
```

Comment:
Why wouldn't it be well organized?

Reply:
Missing Quotes.

Reply:
That's what most of my comments are about, but what I meant was about the warehouse organization: `The warehouse is surprisingly well organized for a place that deals with raw ores.`.

Reply:
```suggestion
			`	The warehouse is surprisingly well organized for a place that deals with raw ores. The receptionist tells you that the owner is due back shortly and points you to some picnic tables outside. You have a seat and watch the trucks loading and carting off various minerals. Soon enough a battered Aerie so covered in scorch marks and hull repairs that you can barely make out its registration markings settles in next to you. When the hatch opens, a woman with close-cut hair and a worn but crisp navy flight suit walks up to you. "I'm Captain Zinaida," she says without preamble. "You were wanting to ask about mining?" You nod your head and she says, "Well, go ahead."`
```

Reply:
> That's what most of my comments are about, but what I meant was about the warehouse organization: `The warehouse is surprisingly well organized for a place that deals with raw ores.`.

This, idk. Why wouldn't it be organised?

Reply:
> > That's what most of my comments are about, but what I meant was about the warehouse organization: `The warehouse is surprisingly well organized for a place that deals with raw ores.`.
> 
> This, idk. Why wouldn't it be organised?

Sorry, I may be communicating badly. That's the question I'm trying to ask.

What about raw ores makes it unlikely for it to be organized?